stix2elevator/convert_cybox.py

```diff
@@ -669,6 +669,38 @@ def convert_attachment(attachment):
     return {"body_raw_ref": attachment.object_reference, "content_type": "text/plain"}
 
 
+_EMAIL_ADDITIONAL_HEADERS_PROPERTIES = {
+    "Message-ID": "message_id",
```

Comment:
I think we have a property in the [EmailMessage ](https://docs.oasis-open.org/cti/stix/v2.1/cs01/stix-v2.1-cs01.html#_grboc7sq5514)to directly capture the `Message-ID`

Reply:
Good catch!!